priv/repo/new_venues.exs

```diff
@@ -76,7 +76,7 @@ defmodule CsGuide.NewVenues do
                 drinks
 
               not_found_drink ->
-                IO.inspect("Drink not found: #{not_found_drink}")
+                IO.inspect("Brand not found: #{not_found_drink}")
```

Comment:
@Danwhy please open an issue in https://github.com/dwyl/learn-elixir/issues to investigate a more _effective_ way of logging errors in an Elixir/Phoenix app. Thanks. ✨